### PR TITLE
ChoiceList

### PR DESCRIPTION
**Pull request by alexbchr**
New ChoiceList component, displaying either `RadioButton`s or `Checkbox`es.

- Add `disabled` prop on `RadioButton` and `Checkbox`.
- Add VS Code snippet to create basic Story structure.

### CODE REVIEW

File: src/components/forms/ChoiceList/ChoiceList.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  error,
+  disabled = false,
+  onChange,
+}) => {
+  const itemProps: ItemsProps[] = choices.map(
+    ({ label, value, disabled: disabledItem, helpText }) => ({
+      key: value,
+      label,
+      checked: selected.indexOf(value) >= 0,
+      disabled: disabledItem || disabled,
```
Comment (hansgx) [inline src/components/forms/ChoiceList/ChoiceList.tsx:76]: Why is this required?

File: src/components/forms/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -30,7 +34,13 @@ const { size } = shameStyles.checkbox
 /**
  * Use when the user needs to choose a single item in a list.
  */
-export const Checkbox: React.FC<CheckboxProps> = ({ label, checked, onChange, helpText }) => {
+export const Checkbox: React.FC<CheckboxProps> = ({
+  label,
+  checked,
+  helpText,
+  disabled = false,
```
Comment (hansgx) [inline src/components/forms/Checkbox/Checkbox.tsx:41]: Small enough change, but going forward I think it would be better to have separate PRs to make sure they stay small and focused! :)

File: src/components/forms/ChoiceList/ChoiceList.tsx (hunk 1 above)
Comment (alexbchr) [reply src/components/forms/ChoiceList/ChoiceList.tsx:76]: We want to disable an `Item` if the user expressly disabled it or it the whole list is disabled. This is why we have the OR condition.

File: src/components/forms/Checkbox/Checkbox.tsx (hunk 2 above)
Comment (alexbchr) [reply src/components/forms/Checkbox/Checkbox.tsx:41]: Of course :slightly_smiling_face:
